filter_rms_error.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def filter_rms_error(filter_object,
+                     to_filter_data_lambda,
+                     desired_filter_data_lambda,
+                     dt=0.01,
+                     start_time=0.0,
```

Comment:
is this in seconds, or in ticks?  If I change the sampling frequency, how is this filter affected?

Reply:
This is in seconds.  If you change the system sampling frequency, then nothing happens because this is just here for testing.